src/cmake/modules/FindLLVM.cmake

```diff
@@ -76,6 +77,9 @@ else ()
 endif ()
 string (REPLACE " " ";" LLVM_SYSTEM_LIBRARIES "${LLVM_SYSTEM_LIBRARIES}")
 
+# Seems necessary on our Windows CI?
+string (REPLACE "libxml2s.lib" "" LLVM_SYSTEM_LIBRARIES "${LLVM_SYSTEM_LIBRARIES}")
```

Comment:
libxml2 is used for llvm-mt, which is a tool for Windows manifest files. Not something that OSL would depend on.

Reply:
Yeah, it seems to end up in the list of libraries that llvm-config reports should be used, but I guess not for our use case.